src/CMakeLists.txt

```diff
@@ -253,29 +253,18 @@ if(MSVC)
     set_target_properties(woof-setup PROPERTIES LINK_FLAGS "/MANIFEST:NO")
 endif()
 
+#
 # Install
+#
 
 # Assemble library files.
 set(WOOF_DLLS "")
+set(VCPKG_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}$<$<CONFIG:Debug>:/debug>/bin")
 if(WIN32 AND NOT MSYS)
-    # SDL2
-    list(APPEND WOOF_DLLS "${SDL2_DLL_DIR}/SDL2.dll")
-
-    # SDL2_mixer
-    list(APPEND WOOF_DLLS "${SDL2_MIXER_DLL_DIR}/optional/libmodplug-1.dll")
-    list(APPEND WOOF_DLLS "${SDL2_MIXER_DLL_DIR}/optional/libopus-0.dll")
-    list(APPEND WOOF_DLLS "${SDL2_MIXER_DLL_DIR}/optional/libopusfile-0.dll")
-    list(APPEND WOOF_DLLS "${SDL2_MIXER_DLL_DIR}/SDL2_mixer.dll")
-
-    # SDL2_net
-    list(APPEND WOOF_DLLS "${SDL2_NET_DLL_DIR}/SDL2_net.dll")
-
-    if (FluidSynth_DLL_DIR)
-      file(GLOB files "${FluidSynth_DLL_DIR}/*.dll")
-      foreach(file ${files})
-        list(APPEND WOOF_DLLS ${file})
-      endforeach()
-    endif()
+    # SDL2_Mixer
+    list(APPEND WOOF_DLLS "${VCPKG_DIR}/modplug.dll")
+    list(APPEND WOOF_DLLS "${VCPKG_DIR}/ogg.dll")
+    list(APPEND WOOF_DLLS "${VCPKG_DIR}/opus.dll")
```

Comment:
We've used the same approach before for MSVC builds. I don't think it's worth complicating this with dynamic dependency lookup, and it's never going to be perfect. For example, what if the next version of SDL_Mixer switches to xmp with modplug?

Reply:
Yes, I agree. We'll always have to keep track of some of these. This isn't going to change anytime soon.